test/unit/spec/LoginRouter_spec.js

```diff
@@ -175,6 +174,17 @@ function (Okta, Q, Backbone, SharedUtil, CryptoUtil, CookieUtil, Logger, OktaAut
       expect(Logger.warn).toHaveBeenCalledWith('Field not defined in schema', arg1);
     }
 
+    var getAutoPushResponse = function (response, autoPushVal) {
```

Comment:
Since this is used in 2 places, would it be worth it to move to some util?

Reply:
It doesn't fit in any of the existing helper utils.. I could create a new util file. What do you think?